src/util/cliffThoughtHeight.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Calculates the height of a drop target at the cliff, decreasing with depth depth. This makes it easier to drop a thought at the desired level. Returns height in ems.
+ */
+const calculateCliffThoughtsHeight = ({ deepestDepth, depth }: { deepestDepth?: number; depth?: number }) => {
+  if (depth === undefined) return 0
+
+  // Calculate dynamic height: increase as depth decreases
+  const baseHeight = 1
+  const heightIncrease = 0.5
+  const dynamicHeight = baseHeight + heightIncrease * ((deepestDepth || depth) - depth)
```

Comment:
Once you rename `deepestDepth` to `cliff` you'll need to flip the sign here to `-cliff` for consistency.

I'm regretting that I originally made cliff negative, but if we're going to flip all the signs we should do that in a separate PR.